### PR TITLE
BAH-2558| Added changeset to update class from Test to LabTest

### PR DESCRIPTION
**Pull request by deepthi-mantena**

### CODE REVIEW

File: masterdata/configuration/liquibase/liquibase.xml [hunk 1]
```diff
@@ -306,4 +306,12 @@
             "org.openmrs.customdatatype.datatype.LongFreeTextDatatype", "org.openmrs.web.attribute.handler.LongFreeTextTextareaHandler", 0, 1, NOW(), 0, "07421454-e196-422d-a7c3-a045ffc5fac9");
         </sql>
     </changeSet>
+    <changeSet id="BAH-2558-17112022" author="Clinic-Config">
+        <comment>Update concepts with class Test to LabTest</comment>
+        <sql>
+            UPDATE concept c, concept_class cc
```
Comment (sivareddyp) [inline masterdata/configuration/liquibase/liquibase.xml]: @deepthi-mantena can you also make sure concept source is `CIEL`

Comment (deepthi-mantena) [reply masterdata/configuration/liquibase/liquibase.xml]: @sivareddytw Updated the change set please review .

File: masterdata/configuration/liquibase/liquibase.xml [hunk 2]
```diff
@@ -306,4 +306,17 @@
...
+    <changeSet id="BAH-2558-17112022-updated" author="Clinic-Config">
+        <comment>Update concepts with class Test to LabTest</comment>
+        <sql>
+            UPDATE concept c
+            JOIN concept_reference_map rmap ON c.concept_id = rmap.concept_id and c.retired = FALSE
+            JOIN concept_reference_term rterm ON rmap.concept_reference_term_id = rterm.concept_reference_term_id
+            JOIN concept_reference_source rsource ON rsource.concept_source_id = rterm.concept_source_id
+            JOIN concept_class cc ON c.class_id = cc.concept_class_id AND cc.name = 'Test'
+            SET
+            c.class_id = (SELECT concept_class_id from concept_class WHERE concept_class.name = 'LabTest')
```
Comment (sivareddyp) [inline masterdata/configuration/liquibase/liquibase.xml]: @deepthi-mantena can you extract concept class Id for `LabTest` to a variable and use it here ?

File: masterdata/configuration/liquibase/liquibase.xml (hunk 1 above)
Comment (deepthi-mantena) [reply masterdata/configuration/liquibase/liquibase.xml]: @sivareddytw extracted concept class Id for LabTest to a variable please verify .

**Review by sivareddyp — DISMISSED** [review]

File: masterdata/configuration/liquibase/liquibase.xml [hunk 3]
```diff
@@ -306,4 +306,19 @@
             "org.openmrs.customdatatype.datatype.LongFreeTextDatatype", "org.openmrs.web.attribute.handler.LongFreeTextTextareaHandler", 0, 1, NOW(), 0, "07421454-e196-422d-a7c3-a045ffc5fac9");
         </sql>
     </changeSet>
+    <changeSet id="BAH-2558-21112022-update" author="Clinic-Config">
```
Comment (sivareddyp) [inline masterdata/configuration/liquibase/liquibase.xml]: @deepthi-mantena liquibase runs before OCL zip in iniz module.. see [this](https://github.com/mekomsolutions/openmrs-module-initializer#supported-domains-and-default-loading-order).
so, for fresh installations, this change set has no effect.

**Review by sivareddyp — APPROVED** [review]

**Review by gsluthra — COMMENTED** [review]
Ok with me. thanks!

**Pull request merged**